pkg/version/util.go

```diff
@@ -0,0 +1,24 @@
+package version
+
+import (
+	"strings"
+)
+
+var (
+	validCurrentVersions = map[string]bool{
+		"1.9.0": true, "1.10.0": true,
+	}
+	validDesiredVersion = GetVersion()
```

Comment:
https://github.com/openebs/cstor-operators/blob/853f02b673948170779407bce1e2f269b0e5f555/pkg/version/version.go#L41
We need to maintain one VERSION file in cstor-operaters repo.

Reply:
yes , the file path has been updated